docs/users/detailed-guides/data-sources/local-files.md

```
@@ -1,537 +1,179 @@
-
 ### Basic Configuration
-
 ```yaml
-global_config:
-  qdrant:
-    url: "http://localhost:6333"
-    collection_name: "documents"
-  openai:
-    api_key: "${OPENAI_API_KEY}"
-
+global:
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)